stdlib/experimental/http/http_experimental.go

```diff
@@ -0,0 +1,120 @@
+package http
+
+import (
+	"context"
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/codes"
+	"github.com/influxdata/flux/internal/errors"
+	"github.com/influxdata/flux/semantic"
+	"github.com/influxdata/flux/values"
+	"github.com/opentracing/opentracing-go"
+	"github.com/opentracing/opentracing-go/log"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+)
+
+// maxResponseBody is the maximum response body we will read before just discarding
+// the rest. This allows sockets to be reused.
+const maxResponseBody = 512 * 1024 // 512 KB
+
+// http get mirrors the http post originally completed for alerts & notifications
+var get = values.NewFunction(
+	"get",
+	semantic.NewFunctionPolyType(semantic.FunctionPolySignature{
+		Parameters: map[string]semantic.PolyType{
+			"url":     semantic.String,
+			"headers": semantic.Tvar(1),
+		},
+		Required: []string{"url"},
+		Return:   semantic.NewObjectPolyType(map[string]semantic.PolyType{"statusCode": semantic.Int, "body": semantic.Bytes, "headers": semantic.String}, semantic.LabelSet{"status", "body"}, nil),
```

Comment:
Can you remove `headers` from the return type here as we are not setting it below anymore?

Reply:
headers in this case is a parameter into the function -  it can be passed in by the user to be built into the request payload